util/libcrypto.num

```diff
@@ -1107,7 +1104,7 @@ OSSL_HTTP_REQ_CTX_new                   1145	3_0_0	EXIST::FUNCTION:HTTP
 X509_load_cert_crl_file                 1146	3_0_0	EXIST::FUNCTION:
 EVP_PKEY_new_mac_key                    1147	3_0_0	EXIST::FUNCTION:
 DIST_POINT_new                          1148	3_0_0	EXIST::FUNCTION:
-BN_is_prime_fasttest                    1149	3_0_0	EXIST::FUNCTION:DEPRECATEDIN_0_9_8
+BN_is_prime_fasttest                    1149	3_0_0	EXIST::FUNCTION:DEPRECATEDIN_0_9_8,DEPRECATEDIN_3_0
```

Comment:
```suggestion
BN_is_prime_fasttest                    1149	3_0_0	EXIST::FUNCTION:DEPRECATEDIN_0_9_8
```
This is what trips up the check update script. It runs `make update` which removes this addition, and hence is unhappy since the repo isn't clean after `make update`. I'll look at the rest now.

Reply:
Oh wow, thanks for the reminder.  I blotted it out of my mind. :). Will fix.